src/ai_marketplace_monitor/facebook.py

```diff
@@ -568,23 +580,38 @@ def get_listing_details(
         return details, False
 
     def check_listing(
-        self: "FacebookMarketplace", item: Listing, item_config: FacebookItemConfig
+        self: "FacebookMarketplace",
+        item: Listing,
+        item_config: FacebookItemConfig,
+        description_available: bool = True,
     ) -> bool:
         # get antikeywords from both item_config or config
         antikeywords = item_config.antikeywords
-        if antikeywords and (
-            is_substring(antikeywords, item.title + " " + item.description, logger=self.logger)
-        ):
-            if self.logger:
-                self.logger.info(
-                    f"""{hilight("[Skip]", "fail")} Exclude {hilight(item.title)} due to {hilight("excluded keywords", "fail")}: {', '.join(antikeywords)}"""
-                )
-            return False
+        if description_available and antikeywords:
```

Comment:
For `antikeywords`, the availability of `description` does not matter, right?

Reply:
I think it does? One issue I realized when looking for a bike was looking for something like

"Sonar 144" (bike) had similar keywords to sound equipment, and antikeywords were critical to be applied against description as well

Reply:
@coderabbitai - do you have opinion on this?

Reply:
@coderabbitai - one thought though - if antikeywords could find a match if the "first pass" with title only, we could prevent uselessly pulling a bad listing description. thoughts on that? Be critical here.

Reply:
Antikeywords should always prevent a listing as soon as one hits. This is the opposite of "keywords" which was our core issue @coderabbitai - I think I'm actually agreeing with @BoPeng here now